editor/sidebar/post-taxonomies/index.js

```diff
@@ -24,58 +24,37 @@ import { toggleSidebarPanel } from '../../actions';
  */
 const PANEL_NAME = 'post-taxonomies';
 
-class PostTaxonomies extends Component {
-	constructor() {
-		super( ...arguments );
+function PostTaxonomies( { postType, taxonomies, isOpened, onTogglePanel } ) {
+	const availableTaxonomies = !! taxonomies.data
+		? Object.values( taxonomies.data ).filter( ( taxonomy ) => taxonomy.types.indexOf( postType ) !== -1 )
```

Comment:
Minor: Lodash's `filter` handles both undefined and Object types, so this could be simplified:

```js
const availableTaxonomies = filter( taxonomies.data, ( { types } ) => includes( types, postType ) );
```